cg/services/orders/order_status_service/order_status_service.py

```diff
@@ -0,0 +1,27 @@
+from cg.apps.tb.api import TrailblazerAPI
+from cg.apps.tb.dto.summary_response import AnalysisSummary
+from cg.services.orders.order_status_service.dto.case_status_summary import CaseSummary
+from cg.services.orders.order_status_service.dto.order_status_summary import OrderSummary
+from cg.services.orders.order_status_service.utils import (
+    add_case_summaries,
+    create_status_summaries,
+    create_case_status_summaries,
+    add_analysis_summaries,
+)
+from cg.store.models import Order
+from cg.store.store import Store
+
+
+class OrderStatusService:
+    def __init__(self, analysis_client: TrailblazerAPI, store: Store) -> None:
+        self.analysis_client = analysis_client
+        self.store = store
+
+    def get_status_summaries(self, order_ids: list[int]) -> list[OrderSummary]:
```

Comment:
It might be the case that this implementation is too in-efficient and that some calculations need to be made into queries instead. I'll test it in stage first